### ISSUE TITLE
Command list not visible in konsole and xterm

### ISSUE THREAD

**Issue by vincentdephily** [adder46/hstr-rs#5]
On konsole and xterm, hstr-rs only clears the terminal and display the prompt and help string. It seems that the rest is working fine, but you're selecting invisible stuff. Tested gnome-terminal and alacritty which work fine.

Some possibly-relevant env vars (didn't try tweaking any) :

    COLORFGBG=15;0
    KONSOLE_DBUS_SERVICE=:1.25
    KONSOLE_DBUS_SESSION=/Sessions/6
    KONSOLE_VERSION=200803
    LANG=en_US.utf8
    SHELL=/bin/bash
    SHLVL=0
    TERM=konsole

hstr-rs v0.1.3 (https://github.com/adder46/hstr-rs.git#2cd65f67)

**Comment by adder46** [comment]
Thanks for reporting the bug.

I'll investigate what is the cause of the problem.

**Comment by adder46** [comment]
Let's start from here:

The issue can be circumvented by appending the following lines to `~/.Xdefaults`:

```
XTerm*internalBorder:  10
XTerm*highlightSelection:  true
XTerm*VT100*colorBDMode:  on
XTerm*VT100*colorBD:  blue
XTerm*VT100*colorULMode:  on
XTerm*VT100*colorUL:  magenta
XTerm.VT100.eightBitInput:  true
XTerm.VT100.eightBitOutput:  true
XTerm*scrollBar:  true
XTerm.VT100.titeInhibit:  true

XTerm.VT100*colorMode:  on
XTerm.VT100*dynamicColors:  on

! Uncomment this to use color for underline attribute
XTerm.VT100*colorULMode:  on
XTerm.VT100*underLine:  off

! Uncomment this to use color for the bold attribute
XTerm.VT100*colorBDMode: on

XTerm.VT100*color0: black
XTerm.VT100*color1: red3
XTerm.VT100*color2: green3
XTerm.VT100*color3: yellow3
XTerm.VT100*color4: blue3
XTerm.VT100*color5: magenta3
XTerm.VT100*color6: cyan3
XTerm.VT100*color7: gray90
XTerm.VT100*color8: gray30
XTerm.VT100*color9: red
XTerm.VT100*color10: green
XTerm.VT100*color11: yellow
XTerm.VT100*color12: blue
XTerm.VT100*color13: magenta
XTerm.VT100*color14: cyan
XTerm.VT100*color15: white
XTerm.VT100*colorUL: yellow
XTerm.VT100*colorBD: white
```
After that, running `xrdb -merge ~/.Xdefaults` is needed for the changes to take effect.

This resolves the issue, but only partially, because another problem occurs, which is `endwin()` does not restore the terminal back to the state before running `hstr-rs`.

**Comment by adder46** [comment]
@vincentdephily, can you check if this works for you?

**Comment by adder46** [comment]
Actually, when I drop to real xterm via CTRL+ALT+F1, it does not work. Hm...

**Comment by vincentdephily** [comment]
Didn't get time to try the `.Xdefaults` workaround, but your commit seems to have fixed it (tested `hstr-rs v0.2.0 (https://github.com/adder46/hstr-rs.git#2454c52e)` in konsole and xterm), thanks.
